### PR TITLE
[4.0] Fieldset is missing a Legendtag

### PR DESCRIPTION
**Pull request by Krshivam25**
Added legend tag within a fieldset .

### CODE REVIEW

**Comment by brianteeman** [comment]
https://www.deque.com/blog/top-5-rules-of-aria/#:~:text=Rule%20%231%3A%20Don't,in%20HTML%2C%20then%20use%20ARIA.

File: administrator/modules/mod_login/tmpl/default.php [hunk 1]
```diff
@@ -39,7 +39,7 @@ class="form-control"
 					required="required"
 					autofocus=""
 					autocomplete="username"
-					aria-required = "true"
+
```
Comment (richard67) [inline administrator/modules/mod_login/tmpl/default.php]: Please remove than empty line.

File: administrator/modules/mod_login/tmpl/default.php [hunk 2]
```diff
@@ -56,7 +56,7 @@ class="form-control"
 					class="form-control input-full"
 					required="required"
 					autocomplete="current-password"
-					aria-required = "true"
+
```
Comment (richard67) [inline administrator/modules/mod_login/tmpl/default.php]: Please remove than empty line.

File: administrator/modules/mod_login/tmpl/default.php [hunk 3]
```diff
@@ -36,8 +37,9 @@
 					type="text"
 					class="form-control"
 					required="required"
-					autofocus
+					autofocus=""
```
Comment (PhilETaylor) [inline administrator/modules/mod_login/tmpl/default.php]: `=""` is not needed here. See https://www.w3schools.com/tags/att_input_autofocus.asp

Comment (Krshivam25) [reply administrator/modules/mod_login/tmpl/default.php]: Thank you for sharing this. I have updated the PR  @PhilETaylor

**Comment by brianteeman** [comment]
Please update the original description

File: administrator/modules/mod_login/tmpl/default.php [hunk 4]
```diff
@@ -53,7 +54,7 @@ class="form-control"
 					type="password"
 					class="form-control input-full"
 					required="required"
-					autocomplete="current-password"
+					autocomplete="current-password"
```
Comment (richard67) [inline administrator/modules/mod_login/tmpl/default.php]:
```suggestion
					autocomplete="current-password"
```
Please remove the trailing tabulator which breaks code style again.

File: administrator/modules/mod_login/tmpl/default.php [hunk 5]
```diff
@@ -24,6 +24,7 @@
 ?>
 <form class="form-validate" action="<?php echo Route::_('index.php', true); ?>" method="post" id="form-login">
 	<fieldset>
+	<legend> <?php echo Text::_('COM_LOGIN'); ?></legend>
```
Comment (Quy) [inline administrator/modules/mod_login/tmpl/default.php]:
```suggestion
	<legend><?php echo Text::_('COM_LOGIN'); ?></legend>
```

File: administrator/modules/mod_login/tmpl/default.php [hunk 6]
```diff
@@ -24,7 +24,7 @@
 ?>
 <form class="form-validate" action="<?php echo Route::_('index.php', true); ?>" method="post" id="form-login">
 	<fieldset>
-	<legend><?php echo Text::_('COM_LOGIN'); ?></legend>
+	<legend class="visually-hidden"> <?php echo Text::_('COM_LOGIN'); ?> </legend>
```
Comment (brianteeman) [inline administrator/modules/mod_login/tmpl/default.php]: please remove those extra spaces

Comment (Krshivam25) [reply administrator/modules/mod_login/tmpl/default.php]: Removed 👍🏻

**Comment by YatharthVyas** [comment]
I have tested this item :white_check_mark: successfully on 856b8b31c9011eb0d0d1766158c581675357883e<br /><br />After PR:

image
<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/34281">issues.joomla.org/tracker/joomla-cms/34281</a>.</sub>

**Comment by rjharishabh** [comment]
I have tested this item :white_check_mark: successfully on 856b8b31c9011eb0d0d1766158c581675357883e<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/34281">issues.joomla.org/tracker/joomla-cms/34281</a>.</sub>

**Comment by richard67** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/34281">issues.joomla.org/tracker/joomla-cms/34281</a>.</sub>

File: administrator/modules/mod_login/tmpl/default.php [hunk 7]
```diff
@@ -24,6 +24,7 @@
 ?>
 <form class="form-validate" action="<?php echo Route::_('index.php', true); ?>" method="post" id="form-login">
 	<fieldset>
+		<legend class="visually-hidden"><?php echo Text::_('COM_LOGIN'); ?></legend>
```
Comment (Quy) [inline administrator/modules/mod_login/tmpl/default.php]: Would it better to use `MOD_LOGIN`?

`MOD_LOGIN="Login Form"`

Comment (richard67) [reply administrator/modules/mod_login/tmpl/default.php]: Good idea

**Comment by Krshivam25** [comment]
I have updated the request. Thank you for noticing this @Quy please test it again @YatharthVyas @rjharishabh

**Comment by rjharishabh** [comment]
I have tested this item :white_check_mark: successfully on 069def98d06a6d79aac289a72126f8de76f51dc7<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/34281">issues.joomla.org/tracker/joomla-cms/34281</a>.</sub>

**Comment by YatharthVyas** [comment]
I have tested this item :white_check_mark: successfully on 069def98d06a6d79aac289a72126f8de76f51dc7<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/34281">issues.joomla.org/tracker/joomla-cms/34281</a>.</sub>

**Comment by richard67** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/34281">issues.joomla.org/tracker/joomla-cms/34281</a>.</sub>

**Comment by Quy** [comment]
Thanks!

**Pull request merged**